pyvirtualcam/camera.py

```diff
@@ -92,19 +97,53 @@ def sleep_until_next_frame(self) -> None:
                 time.sleep(t_sleep)
 
 
-class _WindowsCamera(CameraBase):
+class _NativeCamera(CameraBase):
     def __init__(self, width: int, height: int, fps: float, delay=10, print_fps=False) -> None:
         super().__init__(width, height, fps, delay, print_fps)
-        _native_windows.start(width, height, fps, delay)
+        _native.start(width, height, fps, delay)
 
     def close(self) -> None:
         super().close()
-        _native_windows.stop()
+        _native.stop()
 
     def send(self, frame: np.ndarray) -> None:
         super().send(frame)
-        _native_windows.send(frame)
+        _native.send(frame)
 
 
-if platform.system() == 'Windows':
-    Camera = _WindowsCamera
+class _LinuxCamera(CameraBase):
```

Comment:
This is mostly for API-unification.
Should probably be called "_PyFakeWebcamCamera", so it's re-usable when pyfakewebcam adds more backends.

Reply:
Renamed